app/controllers/api/v1/base_controller.rb

```diff
@@ -35,6 +35,22 @@ def currencies
       render json: Dictionaries.currencies
     end
 
+    def currency_rates
```

Comment:
@dissident why it's in api base controller? There should be no any actions in base controller.

Reply:
@dissident not sure why we ever need this action. All currency calculating should be done on backend side.

Reply:
@dissident why this code is still here?